ui/component/or-app/src/index.ts

```diff
@@ -371,12 +371,16 @@ export class OrApp<S extends AppStateKeyed> extends LitElement {
 
             if (!this._config.logo) {
                 this._config.logo = DefaultLogo;
+            }else{
```

Comment:
Not sure what you mean by this? So should it be the URL for the GET request to the backend, or the location of the file in the images directory? The hardcoded part I can't do much, since `manager.rest` is not initialized by this point in the startup sequence and this is the most central part I can edit these. Any ideas?

Reply:
I don't understand why the URL doesn't contain the correct path in the first place? All paths in the config should be absolute and not require manipulation.

Reply:
I'm not sure I understand what you mean. How are the paths in the config manipulated? The config contains the paths to the images from the images directory in storageDir. The string you see there is simply there to ensure that the correct API endpoint is called. If you want to I can reduce the amount of times we hardcode the string, but I am guessing that's not what you mean.

Reply:
I think this makes the most sense:

When user saves manager config
- All image paths (if they don't already start with ```/api/master/configuration/manager/image/``` then change them to this
- If image doesn't exist in storage dir then check in deployment dir and copy it across otherwise fail save

Frontend should not have to add any prefix to image URLs.